microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1563,12 +1633,18 @@
                           </div>
                       </div>
                   </div>
-                  <p>
+                  <p class="site_specific_content site_specific_content-gut">
                       <strong>{{ _('How can I change my gut microbiome based on my results?') }}</strong>
                   </p>
-                  <p>
+                  <p class="site_specific_content site_specific_content-skin">
+                      <strong>{{ _('How can I change my skin microbiome based on my results?') }}</strong>
```

Comment:
I am worried about these several pieces of text (here and in a few places above) that are exactly the same for skin and gut *except* for the word "skin" or "gut".  I get that when the content is really different, it makes sense to have two different elements, but in cases like this it seems to just be creating a maintenance nightmare.  I would strongly favor incorporating the site as a variable in these pieces of text and then tagging the pieces as belonging to both site_specific_content-skin and site_specific_content-gut so they would be shown in both cases.

Reply:
Ok, that makes sense to me.  I mean, I don't like it, but it makes sense :-/